src/routes/user.py

```
@@ -279,12 +280,15 @@ async def process_standby_list_manually(session: SessionDep, request: Request, b
     if user.status == UserStatus.active: raise HTTPException(409, detail="the user is already active")
     user.status = UserStatus.active
     session.add(user)
-    standbyreq = session.exec(select(StandbyReqTbl).where(StandbyReqTbl.standby_user_id == body.id).where(StandbyReqTbl.is_checked == False)).first()
+    standbyreq = session.exec(select(StandbyReqTbl).where(
+        StandbyReqTbl.standby_user_id == body.id).where(StandbyReqTbl.is_checked == False)).first()
```

Ruff (0.13.3)

Avoid equality comparisons to `False`; use `not StandbyReqTbl.is_checked:` for false checks

Replace with `not StandbyReqTbl.is_checked`

(E712)